internal/service/connect/instance_storage_config_test.go

```diff
@@ -0,0 +1,934 @@
+package connect_test
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/connect"
+	"github.com/hashicorp/aws-sdk-go-base/v2/awsv1shim/v2/tfawserr"
+	sdkacctest "github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	tfconnect "github.com/hashicorp/terraform-provider-aws/internal/service/connect"
+)
+
+//Serialized acceptance tests due to Connect account limits (max 2 parallel tests)
+func TestAccConnectInstanceStorageConfig_serial(t *testing.T) {
```

Comment:
Since there are a maximum of two Connect Instances at a time, this will have to be serialized across all Connect tests that create Instances

Reply:
Resolved in 2315deb817b36c64663f1dc5cb6b1ed07d75b770